doc/storage-examples.md

```diff
@@ -68,5 +68,22 @@ It is advisable to create a dedicated Application Key for Perkeep:
 
 Finally, add the b2 config line to your perkeep `server-config.json`:
 ```
-"b2": "keyID:applicationKey:bucket"
+"b2": "keyID:applicationKey:bucket:endpoint"
 ```
+
+- `endpoint` can be found on the backblaze buckets' interface https://secure.backblaze.com/b2_buckets.htm.
+
+Using low-level configuration:
```

Comment:
I don't think we need to give low-level config examples here if the high-level one works for most users.

This isn't an advanced section of the docs, lets keep it simple.